core/pom.xml

```diff
@@ -207,7 +207,7 @@ THE SOFTWARE.
     <dependency><!-- JENKINS-21160: remoting also depends on args4j, please update accordingly -->
```

Comment:
See this comment

Reply:
See jenkinsci/remoting#329
I'll add you as reviewer there also.